src/Functions/Public/iaas/New-vRAVirtualDisk.ps1

```diff
@@ -214,3 +213,7 @@ function New-vRAVirtualDisk {
 
         }
     }
+
+    New-Alias -Name New-vRABlockDevice -Value New-vRAVirtualDisk
+    Export-ModuleMember -Alias * -Function *
```

Comment:
@chelnak What do you think about this? Should we put any aliases into the module manifest rather than left in the code for a function?

If so, I guess we need to update the build process to get them in the right place automatically.

Reply:
I think a "clean" way to handle aliases is to use an alias attribute:

```
    [CmdletBinding()]
    [Alias("New-vRABlockDevice")]
    PARAM ()
```

Then export them in the module manifest.

Thoughts?

Reply:
I like that idea, seems easy to implement and use

Reply:
I like it 👍🏻 

@Stevio54 Can you update the function to implement the alias in that way and remove the `Export-ModuleMember` command. Then after merging, I'll update the build code in another branch to add any aliases implemented this way into the generated manifest.

Reply:
Ok, on it

Reply:
It is done.